journey-map/public/index.mjs

```diff
@@ -102,18 +108,27 @@ const switchJourney = async (targetJourney, targetState) => {
 };
 
 const setupHeader = () => {
-    // Add header entries for each journey
+    // Add option to journey select for each journey
     Object.entries(JOURNEY_TYPES).forEach(([id, label]) => {
-        const link = document.createElement('a');
-        link.href = getJourneyUrl(id);
-        link.innerText = label;
-        link.onclick = async (e) => {
-            e.preventDefault();
-            await switchJourney(id, null);
+        if (COMMON_JOURNEY_TYPS.indexOf(id) != -1) {
```

Comment:
(it'd probably be easier to iterate over `COMMON_JOURNEY_TYPES` and then look up the label in `JOURNEY_TYPES`)